docs/docker-compose.md

```diff
@@ -20,7 +20,7 @@ services:
     command: --default-storage-engine innodb
     restart: unless-stopped
     healthcheck:
-      test: mysqladmin -pchangemeplease ping -h localhost
+      test: mysqladmin -p$$MYSQL_ROOT_PASSWORD ping -h localhost
```

Comment:
out of curiosity: why are there two $ ?

Reply:
There shouldn't be.

Reply:
Docker Compose can't access the env variables directly in there. I had to try multiple ways. This way was the way that finally worked (stackoverflow helped a lot 😁).

Reply:
Is that the part of the relevant docs: https://docs.docker.com/compose/compose-file/compose-file-v2/#variable-substitution ?

```
You can use a $$ (double-dollar sign) when your configuration needs a literal dollar sign. This also prevents Compose from interpolating a value, so a $$ allows you to refer to environment variables that you don’t want processed by Compose.
```

Reply:
That looks right! Thank you!